### ISSUE TITLE
Space can't be set as leader key

### ISSUE THREAD

**Issue by alexfertel** [tgrosinger/leader-hotkeys-obsidian#33]
Hello!

I use `<space>` as my leader key, which would be nice to have in obsidian as well. However, I can't seem to set it.

**To Reproduce**
1. Try to set `<space>` as the leader key.

**Expected behavior**
Use `<space>` as the leader key.

**Desktop (please complete the following information):**
 - OS: macOS Ventura 13.0
 - Obsidian Version: v1.0.3
 - Leader Hotkeys Version: v0.2.0

**Comment by thor314** [comment]
replicated on popOS linux

**Comment by guysbryant** [comment]
I don't think this is a vim mode issue or even an issue with this plugin. I don't have the option to set <Space> as a leader key in Obsidian itself.

**Comment by rross101** [comment]
I think that [Spacekeys](https://github.com/jlumpe/obsidian-spacekeys) for obsidian does this.
